Source/SwiftLintFramework/Rules/FileLengthRule.swift

```diff
@@ -18,15 +18,20 @@ public struct FileLengthRule: ConfigurationProviderRule, SourceKitFreeRule {
         name: "File Line Length",
         description: "Files should not span too many lines.",
         nonTriggeringExamples: [
-            repeatElement("//\n", count: 400).joined()
+            repeatElement("print(\"swiftlint\")\n", count: 400).joined(),
+            (repeatElement("print(\"swiftlint\")\n", count: 400) + ["//\n"]).joined()
         ],
         triggeringExamples: [
-            repeatElement("//\n", count: 401).joined()
+            repeatElement("print(\"swiftlint\")\n", count: 401).joined()
         ]
     )
 
     public func validate(file: File) -> [StyleViolation] {
-        let lineCount = file.lines.count
+        let commentKinds = Set(SyntaxKind.commentKinds())
+        let lineCount = file.syntaxKindsByLines.filter { kinds in
```

Comment:
probably we should only perform this filtering if the file would trigger a violation.

This reminds me of one of my first PRs and @jpsim's comment: https://github.com/realm/SwiftLint/pull/330#issuecomment-170160772